src/main/java/net/dv8tion/jda/client/JDAClient.java

```diff
@@ -47,4 +47,29 @@
     Friend getFriendById(String id);
 
     UserSettings getSettings();
+
+    /**
+     * Accepts the given invite and joins the guild.
+     *
+     * <br>Possible {@link net.dv8tion.jda.core.requests.ErrorResponse ErrorResponses}:
+     * <ul>
+     * <li>{@link net.dv8tion.jda.core.requests.ErrorResponse#UNKNOWN_INVITE Unknown Invite} <br>
+     * The Message did not exist (possibly deleted)</li>
```

Comment:
if a user is banned discord also returns `UNKNOWN_INVITE`. I'm adding that to the docs